o11y/profiler/datadog/datadog.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package datadog
+
+import (
+	"github.com/DataDog/chaos-controller/o11y/profiler/types"
+	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
+)
+
+// Sink describes a datadog profiler sink
+type Sink struct{}
+
+// New initiated datadog profiler sink
+func New(cfg types.SinkConfig) (Sink, error) {
+	var err error
+
+	if cfg.Enable {
```

Comment:
TBH I would move this outside of the profiler and let the main handle the if enable.

Reply:
Removed the `enabled` flags. I agree that `noop` is the better way to deactivate tracing and profiling.